core/global/plugin_loader.gd

```diff
@@ -100,7 +100,8 @@ func get_plugin_store_items() -> Variant:
 		logger.error("Plugin loader has not been initialized!")
 		return
 	var http: HTTPRequest = HTTPRequest.new()
-	parent.add_child(http)
+	parent.add_child.call_deferred(http)
+	await http.ready
```

Comment:
This is the most common change to adhere to the new thread safety requirements in Godot Engine 4.1. `add_child` can only be called from the main thread. So we defer the call and wait until it is ready.